mlflow/tracking/client.py

```diff
@@ -4355,6 +4361,21 @@ def print_model_version_info(mv):
             mv_copy = client.copy_model_version(src_model_uri, dst_name)
             print_model_version_info(mv_copy)
 
+        .. code-block:: python
+            :caption: Migration example from Databricks Workspace Model Registry to Unity Catalog
+
+            from mlflow import MlflowClient
+            import os
+
+            skip_sig_var = "MLFLOW_SKIP_SIGNATURE_CHECK_FOR_MIGRATION_TO_DATABRICKS_UC_REGISTRY"
+            os.environ[skip_sig_var] = "true"
+            client = MlflowClient(registry_uri="databricks")
+
+            src_model_uri = f"models:/my_wmr_model/1"
+            uc_model_dst_name = "mycatalog.myschema.my_uc_model"
+            uc_migrated_copy = client.copy_model_version(src_model_uri, uc_model_dst_name)
+            print_model_version_info(uc_migrated_copy)
```

Comment:
This isn't a real function, right? Can we print some real properties of the migration copy instance?

Reply:
Yeah this was referencing the function used in the code block above this one, but replaced it with an equivalent print instead to keep things encapsulated